.vscode/settings.json

```diff
@@ -21,6 +21,7 @@
   "prettier.bracketSpacing": false,
   "prettier.singleQuote": true,
   "prettier.printWidth": 80,
+  "prettier.trailingComma": "all",
```

Comment:
Let's discuss.

We are targeting ES2017 (Node.js 8+) by default. To support ES2015 (Node.js 6+), we are transpiling our sources via TypeScript. I believe TypeScript compiler will take care of trailing commas in function arguments and remove them when the target is pre-ES2017.

What are your arguments for not using ES2017 trailing commas in function arguments?